tests/indices/managed/test_google.py

```diff
@@ -11,19 +11,30 @@
 except ImportError:
     has_google = False
 
-from llama_index.indices.managed.google.generativeai import GoogleIndex
+from llama_index.indices.managed.google.generativeai import (
+    GoogleIndex,
+    set_google_auth_credentials,
+)
 
 SKIP_TEST_REASON = "Google GenerativeAI is not installed"
 
 
 if has_google:
     import llama_index.vector_stores.google.generativeai.genai_extension as genaix
 
-    genaix.set_defaults(
+    genaix.set_config(
         genaix.Config(api_endpoint="No-such-endpoint-to-prevent-hitting-real-backend")
```

Comment:
Oh, we should add

                testing=True,

here too!

My bad! Would you mind adding that?

Reply:
done